libs/shared/ui/src/lib/components/slider/slider.spec.tsx

```diff
@@ -0,0 +1,62 @@
+import { render } from '__tests__/utils/setup-jest'
+import { fireEvent } from '@testing-library/react'
+import { Slider, SliderProps } from './slider'
+
+let props: SliderProps
+
+let container
+
+beforeEach(() => {
+  props = {
+    min: 10,
+    max: 100,
+    step: 10,
+  }
+
+  container = document.createElement('div')
+  document.body.appendChild(container)
+})
+describe('Slider', () => {
+  it('should render successfully', () => {
+    const { baseElement } = render(<Slider {...props} />)
+    expect(baseElement).toBeTruthy()
+  })
+
+  it('should apply class name', () => {
```

Comment:
There is no conditionnal logic on the `className` property in the component so no need to test it. Otherwise you will be testing ALL the props passed to a child component which is useless since the framework is already enforcing that for us. 

If you had some piece of code in the component that assigned conditionnally the className property like :

```
const className = className ? className : 'someClassName'
```
Then it would have been ok to test it to increase branching stat